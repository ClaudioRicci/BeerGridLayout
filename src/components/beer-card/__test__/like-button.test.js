import React from "react";
import styled from "styled-components";
import renderer from "react-test-renderer";
import "jest-styled-components";

const LikeButton = styled.button`
  display: block;
  background: transparent;
  border: none;
  cursor: pointer;
  position: absolute;
  top: 20px;
  right: 20px;
`;

test("it works", () => {
  const tree = renderer.create(<LikeButton />).toJSON();
  expect(tree).toMatchSnapshot();
  expect(tree).toHaveStyleRule("background", "transparent");
});
