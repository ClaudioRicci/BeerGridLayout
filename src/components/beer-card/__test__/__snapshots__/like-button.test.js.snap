// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it works 1`] = `
.c0 {
  display: block;
  background: transparent;
  border: none;
  cursor: pointer;
  position: absolute;
  top: 20px;
  right: 20px;
}

<button
  className="c0"
/>
`;
